docs/how-to-start.md

```diff
@@ -49,7 +49,7 @@ following configuration:
 version: "3.9"
```

Comment:
There's another reference that could do with being updated on [line 73](https://github.com/stackrox/collector/blob/mauro/ROX-9802-remove-dockerhub/docs/how-to-start.md?plain=1#L73) of this file

Reply:
how come it's not published to `quay.io/stackrox-io`? 

I think you're right that it makes sense to leave it as is if it would otherwise misguide users.